core-bundle/src/Fragment/FragmentHandler.php

```diff
@@ -113,4 +117,19 @@ private function hasGlobalPageObject(): bool
     {
         return isset($GLOBALS['objPage']) && $GLOBALS['objPage'] instanceof PageModel;
     }
+
+    private function containsNonScalars(array $values): bool
+    {
+        foreach ($values as $value) {
+            if (\is_array($value)) {
+                return $this->containsNonScalars($value);
```

Comment:
If `$values` looks like this `[], object`, this function will return `false`, although the array contains a "non scalar".

Did you mean to write this?
```suggestion
                if ($this->containsNonScalars($value)) {
                      return true;
                }
```

Reply:
well that's a copy of Symfony: https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/HttpKernel/Fragment/AbstractSurrogateFragmentRenderer.php#L89-L100
we should behave like they do – but maybe it is worth fixing in Sf?

Reply:
True, IMHO that's wrong in Symfony as well. I opened an issue: https://github.com/symfony/symfony/issues/46152